.github/workflows/build_main.yaml

```diff
@@ -26,3 +26,9 @@ jobs:
         echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
         make multiarch-build
         docker logout
+     - uses: act10ns/slack@v1
+       with:
+         status: ${{ job.status }}
+         steps: ${{ toJson(steps) }}
+         channel: '#contour-ci-notifications'
+       if: ${{ failure() && github.head_ref == 'refs/heads/main' }}
```

Comment:
nit: might be ok to remove the branch check for this one since its always on main

Reply:
yeah copy/paste just for consistency. I can take out since it's really not needed.